ci-operator/config/quay/quay-tests/quay-quay-tests-master__quay-upgrade.yaml

```diff
@@ -0,0 +1,49 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  ocp_builder_rhel-8-golang-1.20-openshift-4.16:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.16
+  tools:
+    name: "4.16"
+    namespace: ocp
+    tag: tools
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.16
+images:
+- dockerfile_path: images/Dockerfile.upgrade
+  from: tools
+  inputs:
+    ocp_builder_rhel-8-golang-1.20-openshift-4.16:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.16
+  to: quay-test-upgrade
```

Comment:
As this image will be used for Quay Operator testing with golang code, not only for upgrade, suggest use name like "quay-test-operator"

Reply:
updated